composer.json

```diff
@@ -4,20 +4,21 @@
   "license": [
     "BSD-3-Clause"
   ],
-  "version": "2.4.5",
   "require": {
     "php": ">=5.4.0",
-    "nosto/php-sdk": "3.15.0"
+    "nosto/php-sdk": "3.15.*",
+    "ext-json": "*"
   },
   "require-dev": {
-    "php": ">=7.2.0",
-    "squizlabs/php_codesniffer": "2.8.1",
-    "phan/phan": "2.6.*",
+    "phan/phan": "^2.6",
+    "squizlabs/php_codesniffer": "^3.5",
     "phpmd/phpmd": "^2.6",
     "sebastian/phpcpd": "4.1.0",
     "phing/phing": "2.*",
-    "shopware/shopware": "~5.6",
+    "shopware/shopware": "5.5.*",
```

Comment:
You can't downgrade this, will cause a security issue with `symfony/http-foundation (v3.4.27)`

You may want to install symfony security-check locally
`curl -sS https://get.symfony.com/cli/installer | bash`
`mv ~/.symfony/bin/symfony /usr/local/bin/symfony`
`symfony security:check`